lua/nvim-tree.lua

```diff
@@ -636,6 +636,11 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
       trash = true,
     },
   },
+  experimental = {
+    git = {
+      async = false,
+    },
+  },
```

Comment:
It looks like gitsigns has stopped (underscore prefixed) experiments.

Using the same pattern as the various lsp plugins.